test/exportable/ExportableExtension.test.ts

```diff
@@ -98,16 +99,22 @@ describe('ExportableExtension', () => {
     input.rules.push(flagRule);
 
     const bindingRule = new ExportableBindingRule('value[x]');
-    bindingRule.valueSet = 'http://example.org/ValueSet/Foo';
+    bindingRule.
+      = 'http://example.org/ValueSet/Foo';
```

Comment:
```suggestion
    bindingRule.valueSet = 'http://example.org/ValueSet/Foo';
```

Reply:
Thank you for fixing that nasty bad good for nothing merge commit.